### DISCUSSION TITLE
Unterstützung für HmIP-DBB (Taster) vorhanden?

### DISCUSSION THREAD

**Discussion by Attack44** [SukramJ/aiohomematic#942] [Q&A] [answered]
Hallo,

wie sieht es mit der Unterstützung/Einbindung des Klingeltasters (HmIP-DBB) aus?
Ich habe den Taster in RasberryMatic hinzugefügt und an sich ist dieser auch in Home Assistant sichtbar, aber ich habe keine Entität für den eigentlichen Taster, nur für den Batteriestatus.

Im Logbuch wird mir auch im HA angezeigt wenn ich den Taster drücke, aber ohne Entität kann ich ihn z.B. in Node-RED nicht verwenden oder? Für Tipps wäre ich sehr dankbar. :)

Gruß
Andreas

![Screenshot 2023-02-08 213709](https://user-images.githubusercontent.com/19206966/217645453-aeba64f4-fad4-4daf-b513-a21d853caf6d.png)

**Comment by SukramJ** [comment, marked as answer]
Ja, einfach auf Automatisierung klicken und das gewünschte Ereignis auswählen.

**Comment by Baxxy13** [comment]
Irgendwie ist das ja auch ein bisschen "doppelt-gemoppelt".

RM/CCU senden an HA, HA soll "weiterleiten" an NodeRed wo dann die Automatismen stattfinden?

Da würde ich doch entweder:

- wie vorgeschlagen die Automatisierung direkt in HA machen
- NodeRed direkt an die CCU / RM koppeln (ohne Umweg über HA) um dort direkt auf die Events reagieren zu können

**Comment by Attack44** [comment]
Einfach auf "Automatisierung" zu klicken bin ich nicht gekommen...war wohl zu einfach. Danke. 🙈 

@Baxxy13 Ja das stimmt, es ist ein wenig doppelt-gemoppelt, aber mir war/ist bisher nicht bekannt wie ich im Node-Red Addon im HA direkt auf RM zugreifen kann ohne über HA zu gehen.

**Reply by Baxxy13** [reply]
[@ptweety/node-red-contrib-ccu](https://flows.nodered.org/node/@ptweety/node-red-contrib-ccu)

**Reply by ThomDietrich** [reply]
Nett dass das geht, allerdings sehe ich Home Assistant als das zentrale System in meinem Heim. Ich würde daher so wie Attack44 bisher auch keine Dreiecksabhängigkeit eingehen wollen. Ich würde sogar noch einen Schritt weiter gehen und behaupten, dass wenn das verlinkte Node Red Paket sinnvolle CCU Fähigkeiten bereistellt, welche einen echten mehrwert bieten, dann sollten mittelfristig diese hier bzw in HA nachgerüstet werden :)

**Reply by Baxxy13** [reply]
Grundsätzlich bin ich auch der Meinung das es nur "eine" zentrale übergeordnete Instanz geben sollte.
Von den Fähigkeiten gibt es zwischen NodeRed (node-red-contrib-ccu) / ioBroker (HomeMatic ReGaHSS + HomeMatic RPC - Adapter) und HA (Homematic(IP) Local - Integration) bezüglich "CCU - Anbindung" kaum noch Unterschiede.

ioBroker und NodeRed können noch CUxD, NodeRed zusätzlich CCU-Jack Devices. Das wären Vorteile falls man das benötigt.

Die Dreiecksbeziehung brachte ja @Attack44 ins Spiel, weil er (mangels Wissen?) den von HA empfangenen CCU-Tastendruck mittels NodeRed "weiterverarbeiten" wollte. Kann ja sein das was mit NodeRed gesteuert werden soll weil das in/mit HA nicht möglich ist.
In dem Fall würde ich eben NodeRed direkt mit der CCU verbinden und keinen Abstecher über HA machen. :wink:
